File: PYTHON/KODEPROGRAM/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
    def __init__(self, id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna):
        super().__init__(id, nama_produk, harga_produk, stok_produk)
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna

    def get_jenis(self):
        return self.jenis

    def set_jenis(self, jenis):
        self.jenis = jenis

    def get_bahan(self):
        return self.bahan

    def set_bahan(self, bahan):
        self.bahan = bahan

    def get_warna(self):
        return self.warna

    def set_warna(self, warna):
        self.warna = warna

File: JAVA/KODEPROGRAM/Aksesoris.java
class Aksesoris extends Petshop {
	
	private String jenis;
	private String bahan;
	private String warna;
	
	public Aksesoris(){
	}
	
	public Aksesoris(String jenis, String bahan, String warna){
		this.jenis = jenis;
		this.bahan = bahan;
		this.warna = warna;
	}
	
	public void setJenis(String jenis){
		this.jenis = jenis;
	}
	public String getJenis(){
		return jenis;
	}

	public void setBahan(String bahan){
		this.bahan = bahan;
	}
	public String getBahan(){
		return bahan;
	}

	public void setWarna(String warna){
		this.warna = warna;
	}
	public String getWarna(){
		return warna;
	}

}